assets/css/nav/_top_nav.scss

```diff
@@ -52,3 +53,17 @@
     color: $color-eggplant-800;
   }
 }
+
+.c-top-nav__user-icon {
+  background-color: #6d39ac;
+  color: #ffffff;
+  height: 24px;
+  width: 24px;
+  font-size: 15px;
+  line-height: 24px;
+  border-radius: 12px;
+}
+
+.c-top-nav__popover {
+  z-index: 2000;
```

Comment:
**suggestion:** I think this could be an entry in the `$z-page-layout-context` map in `app.scss` (sorry, I feel like I may have commented this on an earlier iteration of this PR and it got lost maybe by being part of a commit comment rather than PR comment or something).

Reply:
I think you did - thanks for re-raising!

Reply:
This seems to work? https://github.com/mbta/skate/pull/2385/commits/b1965fb634919a266e099dc680b0f2b23a26f8e8